__tests__/app.test.js

```diff
@@ -212,23 +234,44 @@ describe("POST", () => {
         expect(body.msg).toBe("status 400: invalid request, missing information");
       })
   })
-  test("GET 404: responds with message and error if input article_id does not have an article attached", () => {
-    return request(app)
-      .get("/api/articles/300/comments")
+  test("POST 404: responds with invalid request error message when post request is sent with an invalid username", () => {
+    const testInput = {
+      username: "hacker",
+      body: "really important words about thoughts"
+    }
+      return request(app)
+      .post("/api/articles/1/comments")
+      .send(testInput)
       .expect(404)
       .then(({ body }) => {
-        expect(body.msg).toBe("status 404: Article not found");
+        expect(body.msg).toBe("status 404: not found");
+      })
+  })
+  test("POST 400: responds with message and error if input article_id is valid but does not have an article attached", () => {
+    return request(app)
+      .post("/api/articles/300/comments")
+      .expect(400)
```

Comment:
Really nice test! This would however be an error code of `404` ( valid but not found in the database ) , `400` would be for if the ID is invalid such as in your next test